### DISCUSSION TITLE
[Projects Beta] Missing activity list

### DISCUSSION THREAD

**Discussion by voltaek** [community/community#17270] [Projects and Issues]
I'm concerned with the lack of an activity list in the this new project tracker. In the current, non-beta version of projects, the activity tracker is the main way I keep up what is happening in the repos that I follow and contribute to. It's the easiest way to see what has happened over the last X number of days so I can stay updated and participate accordingly. In that version, the activity list is in the right-side slide-out labeled "Menu". The slide-out has the description of the project at the top and a scrollable activity list below it, with a "More" button at the bottom to load older activity in-place. The beta version has a similar slide-out from the right side, but an activity list is notably absent.

With how much I use the activity lists, I have always wished for more features on them (such as filtering by labels, status, timeframe, etc or even just being able to open the issue/PR directly without having to go its card to click the card's link) in order to make looking through the history activity easier and more efficient, but to see it go the opposite way and be completely dropped from the project tracker revamp is an oversight, in my opinion. Please migrate at least the current simple list version of it, if not a version with some basic filtering.

**Comment by matanhavi** [comment]
Thumb up.
And addon, In the issue itself there is no history of the status like it in regular project:
![image](https://user-images.githubusercontent.com/1760878/175822931-0a965f52-25f6-44c9-a372-116f982d2f32.png)
So I can't see who from the team changed stauts of the issue. for example, who decided that the issue is closed
